wren-ui/src/apollo/server/schema.ts

```diff
@@ -366,6 +366,8 @@ export const typeDefs = gql`
     description: String
     isPrimaryKey: Boolean!
     expression: String
+    aggregation: String
+    lineage: [Int!]
```

Comment:
The lineage column in the DB is a nullable column
Only the calculated field has lineage, so we couldn't make it required, it will cause GraphlQL error.

Reply:
Thanks for explaining it. : D